pkgs/top-level/php-packages.nix

```diff
@@ -4,7 +4,7 @@ let
   self = with self; {
     buildPecl = import ../build-support/build-pecl.nix {
       inherit php;
-      inherit (pkgs) stdenv autoreconfHook fetchurl re2c;
+      inherit (pkgs) stdenv autoreconfHook fetchurl file re2c;
```

Comment:
I question the need of file here as well.

Reply:
Without it, such a this error
`error: anonymous function at /home/user/works/src-nix/nixpkgs/pkgs/build-support/build-pecl.nix:1:1 called without required argument 'file', at /home/user/works/src-nix/nixpkgs/pkgs/top-level/php-packages.nix:5:17`

Reply:
That error occurs only because you added it as a required argument for that function. But in another comment I questioned if it was required. Because if it's not required it could be removed from the function as well as an input to the function here.

Reply:
Аrgument 'file' need on function build-pecl.